### DISCUSSION TITLE
Manual synchronize on Ubuntu

### DISCUSSION THREAD

**Discussion by PurnenduKarmakar** [abraunegg/onedrive#1291] [Q&A] [answered]
Appreciate your hard work. First day of OneDrive on Ubuntu 20.04 (64bit). 
1) Does Onedrive keep running in the background of a session if I run "onedrive --synchronize" ? (I rarely use OneDrive. Is there a way to Sync manually ONLY when I need, and avoid losing memory normally) 
2) Why do I see "onedrive -m" running all the time in the background? 

(Note that I tried sudo docker stop onedrive and without sudo, did not work)

**Comment by abraunegg** [comment, marked as answer]
@fullmoonyogi 
> Does Onedrive keep running in the background of a session if I run "onedrive --synchronize" ? (I rarely use OneDrive. Is there a way to Sync manually ONLY when I need, and avoid losing memory normally)

If the session remains active, then yes, if you close the session - then the sync will terminate

> Why do I see "onedrive -m" running all the time in the background?

This is because you are running Ubuntu and installed the client via the PPA. This is a *packaging* issue only. There are 2 ways to solve this:
1. Fix the package by emailing the maintainer of the packge
2. Uninstall the PPA package and build from source

See: https://github.com/abraunegg/onedrive/issues/1274 for further details
